### DISCUSSION TITLE
Image gallery

### DISCUSSION THREAD

**Discussion by lucamondini** [jpanther/congo#417] [Q&A] [answered]
Hi and thank you for the awesome theme.
Which is the best way to display an image gallery in a post or page?
A list of 'figure' shortcodes? Are there other ways?
Thank you so much.

**Comment by jpanther** [comment, marked as answer]
A gallery is not currently directly supported by the theme but you could look at something like [mfg92/hugo-shortcode-gallery](https://github.com/mfg92/hugo-shortcode-gallery)
